<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentDialog</class>
 <widget class="QDialog" name="StudentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="middleNameLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Mid Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="birthDateLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Birth Date</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>400</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lastNameLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="facultyLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Faculty</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneNumberLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone Number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstNameLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="courseLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Course</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="groupLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Group</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
